libs/testing/src/user.session.ts

```diff
@@ -121,8 +121,6 @@ export class UserSession {
       await this.addOrganization();
     }
 
-    await this.fetchJWT();
```

Comment:
I think it is needed because after adding an organization we need to generate a new jwt token with organizationId, because at this point it won't exist.

Reply:
Will add it back.